src/notation/inotationplayback.h

```diff
@@ -56,6 +56,7 @@ class INotationPlayback
 
     virtual RectF playbackCursorRectByTick(midi::tick_t tick) const = 0;
 
+    virtual RetVal<midi::tick_t> playPositionTickByRawTick(midi::tick_t tick) const = 0;
```

Comment:
The purpose of this method is quite unclear. What is the difference between "regular" and "raw" ticks?

Reply:
It's basically a wrapper around `score()->repeatList().tick2utick(tick)`, which converts from "ticks without considering repeats" (i.e. just incrementing linearly from start to end of score) to "ticks with considering repeats" (i.e. unrolling all repeat sections before `tick`). Do you have a better idea for a name?